apis/python/src/tiledbsoma/_arrow_types.py

```diff
@@ -50,6 +50,24 @@
     pa.date32(): TypeError("32-bit date - unsupported type (use TimestampType)"),
     pa.date64(): TypeError("64-bit date - unsupported type (use TimestampType)"),
 }
+
+_PYARROW_TO_CARROW: Dict[pa.DataType, str] = {
+    pa.bool_(): "b",
+    pa.int8(): "c",
+    pa.int16(): "s",
+    pa.int32(): "i",
+    pa.int64(): "l",
+    pa.uint8(): "C",
+    pa.uint16(): "S",
+    pa.uint32(): "I",
+    pa.uint64(): "L",
+    pa.float32(): "f",
+    pa.float64(): "g",
+    pa.timestamp("s"): "tss:",
+    pa.timestamp("ms"): "tsm:",
+    pa.timestamp("us"): "tsu:",
+    pa.timestamp("ns"): "tsn:",
+}
 """Dict of types unsupported by to_pandas_dtype, which require overrides for
 use in TileDB Attributes (aka DataFrame non-indexe columns).
```

Comment:
This comment has been separated from its home :(

Reply:
Fixed now but just curious -- I think this is the second time in this codebase where the documentation is beneath the data struct instead of above it. Is that typical?